server/data/visitSchedulerApiClient.test.ts

```diff
@@ -63,33 +62,7 @@ describe('visitSchedulerApiClient', () => {
   describe('getVisit', () => {
     it('should return a single matching Visit from the Visit Scheduler API for a valid reference', async () => {
       const reference = 'ab-cd-ef-gh'
-      const result: Visit = {
-        applicationReference: 'aaa-bbb-ccc',
-        reference: 'ab-cd-ef-gh',
-        prisonerId: 'A1234BC',
-        prisonId: 'HEI',
-        visitRoom: 'A1 L3',
-        visitType: 'SOCIAL',
-        visitStatus: 'RESERVED',
-        visitRestriction: 'OPEN',
-        startTimestamp: timestamp,
-        endTimestamp: '',
-        visitNotes: [],
-        visitors: [
-          {
-            nomisPersonId: 1234,
-          },
-        ],
-        visitorSupport: [
-          {
-            type: 'OTHER',
-            text: 'custom support details',
-          },
-        ],
-        createdBy: 'user1',
-        createdTimestamp: '2022-02-14T10:00:00',
-        modifiedTimestamp: '2022-02-14T10:05:00',
-      }
+      const result = TestData.visit()
```

Comment:
I've been able to remove some excess test data by using `TestData`. In other places I've left it for another time and just tweaked to avoid unpicking too much.